support-frontend/assets/components/headers/links/links.tsx

```diff
@@ -115,10 +115,14 @@ function Links({
 			<ul className="component-header-links__ul" ref={getRef}>
 				{links
 					.filter(({ text }) => {
+						const internationalisationIDValue =
```

Comment:
Currently we're setting `internationalisationIDValue` for each item in the `links` array as we iterate over it, however it's value will always be the same so we may as well only set it once, we could move the variable definition to the top of the `Links` function along with the other variable definitions such as `urlWithoutParams` and only define it once.

Reply:
We could also just save a variable `const isNotUk = internationalisationIDValue !== 'uk'` and then we wouldn't have to duplicate the conditional check `internationalisationIDValue !== 'uk'` multiple times.

Reply:
Yes agree, much better than repeating the internationalisationID call per links item and isNotUK more readable especially. Have just put the changes in. Just a point of note, with ab-newProduct. When routing/clicking  between links they always go back to the non-variant (ie flipping between UK Subscription/NewsPaper/GWeekly always goes to the non-variant version link). When testing would the UI guys want this? Wish had asked earlier now!

Reply:
I think when it comes to sharing links for Design/UX to preview the changes it'd be best to create a branch where we enable the `newProduct` AB test and deploy it to `CODE`.